activesupport/test/time_zone_test.rb

```diff
@@ -8,6 +8,17 @@
 class TimeZoneTest < ActiveSupport::TestCase
   include TimeZoneTestHelpers
 
+  def setup
+    ActiveSupport::TimeZone.instance_variable_set(:@lazy_zones_map, Concurrent::Map.new)
```

Comment:
This setup is overly coupled to the internal details of the implementation

Reply:
Do you have any suggestions for an alternative way to do this? The test I wrote depends on the caches being cleared before making the first call to `ActiveSupport::TimeZone.all`.